components/forms/banks/BankSearch.vue

```diff
@@ -66,49 +82,41 @@
           </ListPicker>
         </div>
       </transition>
-    </div>
-  </div>
-  <div v-else class="relative flex items-center">
-    <div
-      class="relative w-full border border-gray-50 text-gray-400 bg-gray-50 rounded-xl shadow-sm pl-12 pr-10 py-4 text-left cursor-default sm:text-sm truncate select-none"
```

Comment:
handled this within the `placeholder` props directly. It's unnecessary to have a separate absolute div that is placed

Reply:
The loader is also placed rather where also the other bank icon is placed